feincms3_cookiecontrol/static/feincms3_cookiecontrol/style.css

```diff
@@ -157,22 +157,48 @@
   width: 0;
 }
 
-.f3cc label::before {
+.f3cc-category label::before {
   content: " ";
+  cursor: pointer;
+  width: 48px;
```

Comment:
@matthiask eventuell alles in rem umrechnen, dann passt sich alles der jeweiligen Seite an?

Reply:
Ja, Du kannst mit dem Button hier auch direkt Vorschläge machen inline :)

![image](https://user-images.githubusercontent.com/2627/140288482-5dacc268-3515-464f-9c1e-0984fcc6e91f.png)